sunpy/net/solarnet/solarnet.py

```diff
@@ -0,0 +1,161 @@
+import os
+import json
+import pathlib
+
+import requests
+
+from sunpy.net import attrs as a
+from sunpy.net.attr import and_
+from sunpy.net.base_client import QueryResponseTable
+from sunpy.net.dataretriever import GenericClient
+from sunpy.net.solarnet.attrs import Dataset, Tags, Target, walker
+
+_BASE_URL = "https://solarnet2.oma.be/service/api/svo/{}"
+
+__all__ = ["SOLARNETClient"]
+
+class SOLARNETClient(GenericClient):
```

Comment:
I find it odd that this is a subclass of `GenericClient` but isn't using any of the scraping functionality that client implements. As far as I see it this class is only using the fetch method of GenericClient?

My recommendation (although I wonder what @nabobalis and anyone else thinks) would be to have this subclass BaseClient instead and reimplement the fetch method (copying it from GenericClient).